src/celeritas/global/alongstep/AlongStepRZMapFieldMscAction.cu

```diff
@@ -41,12 +46,24 @@ void AlongStepRZMapFieldMscAction::execute(CoreParams const& params,
             *this, "propagate-rzmap");
         launch_kernel(params, state, *this, execute_thread);
     }
-    detail::launch_apply_msc(
-        *this, msc_->ref<MemSpace::native>(), params, state);
-    detail::launch_update_time(*this, params, state);
-    detail::launch_apply_eloss(
-        *this, fluct_->ref<MemSpace::native>(), params, state);
-    detail::launch_update_track(*this, params, state);
+    {
+        ScopedProfiling profile_this{label() + "-apply-msc"};
```

Comment:
These could probably be more readable inside `detail::launch...` and we might only want to add the extra granularity for the more expensive kernels?

Reply:
> only want to add the extra granularity for the more expensive kernels?

I was considering that as well, since we can chose which range to profile I figured I might as well add the option to profile any kernel but we'd probably only ever need the propagate